### ISSUE TITLE
how to get dev environment setup?

### ISSUE THREAD

**Issue by lancejpollard** [matijapretnar/eff#14]
Eff looks really cool, very excited to check it out. Trying to get it building locally on OSX by following the README, but running into an issue:

```
$ ./configure
E: Field 'findlib_version' is not set: Field 'ocamlfind' is not set
```

How do you set those variables? Google didn't give anything, though https://ocaml.org/docs/install.html#Debian shows:

```
apt-get install ocaml-findlib
```

Is it required to install `findlib` on the Mac? How did you get it working, are you running on Linux or a Mac?

I got ocaml setup with homebrew:

```
$ brew install ocaml
$ brew install opam
$ brew install menhir
$ ocaml -version
The OCaml toplevel, version 4.02.1
```

Any ideas?

**Comment by lancejpollard** [comment]
Missing step was `opam install batteries`. So in total:

```
$ brew install ocaml
$ brew install opam
$ brew install menhir
$ opam install batteries
$ ./configure
```

**Comment by matijapretnar** [comment]
This is strange: `batteries` was never a requirement. Maybe new OPAM doesn't install `ocamlfind` by default? Thank you for finding a temporary solution, but I guess I need to look into this.

**Comment by andrejbauer** [comment]
Batteries is an overkill. I am prettu sure `ocamlfind` is not there by default on new opam.

**Comment by matijapretnar** [comment]
This will be resolved by #15. Thanks @yallop!
